CHANGES

```diff
@@ -56,6 +81,16 @@ Version 8.1.0 - In development
     - Fixed a memory leak in the OpenVDB AX parser
     - Fixed an occurrence of undefined behaviour in the OpenVDB AX visitor
     - Fixed some memory leaks in the OpenVDB and OpenVDB AX unit tests
+    - Fixed a bug which could cause string allocations in loops to overflow the
+      stack
```

Comment:
Sorry to make you update this again, but the wording I used here is bad in the context of all release notes. It might make sense to have an AX section. Regardless, can we change this to, "Fixed a bug in **AX** which..." ? Likewise in changes.txt. I'm happy to do this after this is merged.

Reply:
Oh woops, this has been merged. I'll put in a PR.